### PR TITLE
kernel-arch-complete.sh: Add weiny2 tree reports emails

### PR DESCRIPTION
**Pull request by nuclearcat**
As requested by https://github.com/kernelci/kernelci-core/issues/1120
Related to https://github.com/kernelci/kernelci-core/pull/1154

Signed-off-by: Denys Fedoryshchenko <denys.f@collabora.com>

### CODE REVIEW

**Review by gctucker — COMMENTED** [review]
It's OK but grouping the recipients on a same line would make it easier to maintain.

File: scripts/kernel-arch-complete.sh [hunk 1]
```diff
@@ -310,7 +310,12 @@ fi
         echo "Sending results for vireshk's tree"
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'", "build_report": 1, "format": ["txt"], "send_to": ["vireshk@kernel.org"], "delay": 60}' ${API}/send
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "baseline", "send_to": ["vireshk@kernel.org"], "format": ["txt"], "delay": 2700}' ${API}/send
-
```
[the remark this answers is not in the document]
Comment (nuclearcat) [reply scripts/kernel-arch-complete.sh:313]: Updated, please let me know if any problem remains

File: scripts/kernel-arch-complete.sh [hunk 2]
```diff
@@ -310,7 +310,12 @@ fi
...
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'", "build_report": 1, "format": ["txt"], "send_to": ["ira.weiny@intel.com"], "delay": 60}' ${API}/send
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'", "build_report": 1, "format": ["txt"], "send_to": ["iweiny@gmail.com"], "delay": 60}' ${API}/send
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "baseline", "send_to": ["ira.weiny@intel.com"], "format": ["txt"], "delay": 2700}' ${API}/send
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "baseline", "send_to": ["iweiny@gmail.com"], "format": ["txt"], "delay": 2700}' ${API}/send
```
[the remark this answers is not in the document]
Comment (nuclearcat) [reply scripts/kernel-arch-complete.sh]: Grouped emails, just didnt noticed it is supposed. P please let me know if any problem remains

**Review by gctucker — APPROVED** [review]
Tested OK on staging (no side-effects with staging email reports)

**Pull request merged**